src/components/Form/ChainConfigurationForm.tsx

```diff
@@ -459,6 +453,28 @@ export const ChainConfigurationForm = withStyles(styles)(
                             />
                           </Grid>
 
+                          <Grid item xs={12} md={6}>
+                            <Field
+                              component={TextField}
+                              id="ocr2P2PPeerID"
+                              name="ocr2P2PPeerID"
+                              label="Peer ID"
+                              select
+                              required
```

Comment:
Should we make this required? For CLO, this is not required, so I'm a bit concerned on any impact there

Reply:
Yeap i had the same concern, but @giogam mentioned it should be required. My limited understanding also tells me that each node will always have a peerId , is that correct?

Reply:
From what I know, a node will always have at least one p2pbundle so it should be safe to require one to be set? Anyway we can probably set this as optional if there are concerns

Reply:
Updated the field to be optional